documentation/src/docs/asciidoc/release-notes/release-notes-5.12.0-M1.adoc

```diff
@@ -86,6 +86,11 @@ JUnit repository on GitHub.
   a test-scoped `ExtensionContext` in `Extension` methods called during test class
   instantiation. This behavior will become the default in future versions of JUnit.
 * `@TempDir` is now supported on test class constructors.
+* `@ParameterizedTest`s now support argument count validation.
```

Comment:
I haven't found any other release note referencing whether it's experimental, so I haven't included it here, either.

Reply:
```suggestion
* Parameterized tests now support argument count validation.
```